src/backend/gl/src/queue.rs

```diff
@@ -207,6 +213,22 @@ impl CommandQueue {
         let gl = &self.share.context;
         let extent = swapchain.extent;
 
+        // Wait for rendering to finish
+        unsafe {
+            let wait_result = gl.client_wait_sync(
```

Comment:
this actually needs to be `wait_sync`, not `client_wait_sync`

Reply:
Interesting, glow doesn't actually have a `wait_sync`. :astonished: 

https://docs.rs/glow/0.6.0/glow/?search=wait_sync

Should I open an issue for glow on that? Is it maybe because it isn't supported on GL ES or something? Maybe that is just a simple PR to glow I could make?

Reply:
Yes, I recommend implementing it quickly in a fork, making a PR (it's a non-breaking change), and using your fork for the dependency here in the meantime.

Reply:
OK, updated glow with the extra binding and submitted a PR and updated this branch with the changes, but still no change in rendering.

Reply:
looks like this still needs to be done?